### PR TITLE
Extern api integration

### PR DESCRIPTION
**Pull request by alisonreynoso**
## Summary
- Completed feature: search user by isbn
## Trello Card
https://trello.com/c/ioD2SlvB/22-integraci%C3%B3n-api-externa

### CODE REVIEW

**Review by julianvasquez-wolox — APPROVED** [review]
Good job! keep it up 🙌🏼

File: src/main/java/wolox/training/dtos/BookDto.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BookDto {
+
+    private String title;
+    private String subtitle;
+    private List<AuthorDto> authors;
+    private int number_of_pages;
+    private String pagination;
+    private List<PublisherDto> publishers;
+    private String publish_date;
```
Comment (Jimenarosello) [inline src/main/java/wolox/training/dtos/BookDto.java]: same here 😅

File: src/main/java/wolox/training/dtos/BookDto.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BookDto {
+
+    private String title;
+    private String subtitle;
+    private List<AuthorDto> authors;
+    private int number_of_pages;
```
Comment (Jimenarosello) [inline src/main/java/wolox/training/dtos/BookDto.java]: We usually name object attributes with camelCase rather than snake_case 🙂

**Review by Jimenarosello — APPROVED** [review]
Nice! 😊

File: src/main/java/wolox/training/dtos/BookDto.java (hunk 2 above)
Comment (alisonreynoso) [reply src/main/java/wolox/training/dtos/BookDto.java]: Done :D x2 (?

**Pull request merged**